reminder/module.py

```diff
@@ -0,0 +1,465 @@
+import asyncio
+import re
+
+from datetime import datetime, timedelta
+from typing import Optional, List
+
+import nextcord
+from nextcord.ext import commands, tasks
+from nextcord.errors import HTTPException, Forbidden
+
+import dateutil.parser
+
+from pie import check, i18n, logger, utils
+
+from .database import ReminderStatus, ReminderItem
+
+_ = i18n.Translator("modules/reminder").translate
+bot_log = logger.Bot.logger()
+guild_log = logger.Guild.logger()
+
+
+class Reminder(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.reminder.start()
+
+    def cog_unload(self):
+        self.reminder.cancel()
+
+    @tasks.loop(seconds=30)
+    async def reminder(self):
+        max_remind_time = datetime.now() + timedelta(seconds=30)
+
+        items = ReminderItem.get_all(
+            status=ReminderStatus.WAITING, max_remind_date=max_remind_time
+        )
+
+        if items is not None:
+            for item in items:
+                await self._remind(item)
+
+    @reminder.before_loop
+    async def before_reminder(self):
+        print("Reminder loop waiting until ready().")
+        await self.bot.wait_until_ready()
+
+    @commands.guild_only()
```

Comment:
Don't make this guild_only. It's not necessary

Reply:
It is 'cause all other commands are guild related. You would not be able to manipulate with reminder created in DM's.

I asked sinus-x about this (makind reminder bonded to guilds) and he wanted it like that.